crypto/src/core/rgsw/method.rs

```diff
@@ -40,6 +43,47 @@ pub fn sk_encrypt<'a, 'b, 'c, R, T>(
     });
 }
 
+pub fn pk_encrypt<'a, 'b, 'c, R: RingOps>(
```

Comment:
is this used anywhere? Just curious

Reply:
I think it'd be used in interactive key gen?